src/ripple/consensus/Consensus.cpp

```diff
@@ -127,7 +140,12 @@ checkConsensus(
                     << prevProposers << " agree=" << currentAgree
                     << " validated=" << currentFinished
                     << " time=" << currentAgreeTime.count() << "/"
-                    << previousAgreeTime.count();
+                    << previousAgreeTime.count()
+                    << " proposing? " << proposing
+                    << " minimum duration to reach consensus: "
+                    << parms.ledgerMIN_CONSENSUS.count() << "ms"
+                    << " max consensus time " << parms.ledgerMAX_CONSENSUS.count() << 's'
```

Comment:
`ledgerMAX_CONSENSUS` type is milliseconds, not seconds.

Reply:
fixed